css-display-3/display-contents-alignment-001-ref.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Test Reference</title>
+<link rel="author" title="Emilio Cobos Álvarez" href="mailto:ecobos@igalia.com">
+<style>
+  .container {
+    display: flex;
+    width: 300px;
+    height: 300px;
+    align-items: center;
+  }
+  .container > div {
+    width: 100px;
+    height: 100px;
+    background: blue;
+  }
+</style>
```

Comment:
It's always nice to add a sentence like:
> Test passes if there is a filled green square vertically centered inside the box.

For that we'd need to add a border to flex container.